purview/models/errata.py

```diff
@@ -47,3 +47,14 @@ class AdvisoryState(PurviewStructuredNode):
     name = StringProperty(required=True)
     advisory = RelationshipTo('Advisory', 'STATE_OF', cardinality=ZeroOrOne)
     creator = RelationshipTo('.user.User', 'CREATED_BY', cardinality=ZeroOrOne)
+
+
+class Product(PurviewStructuredNode):
+    """Definition of a released Product"""
+
+    associated_koji_tag = RelationshipTo('.koji.KojiTag', 'ASSOCIATED_TO')
```

Comment:
This needs to be plural because there can be more than one Koji tag associated with an Errata product.

Reply:
The relationship name should be `ASSOCIATED` for consistency with the others.

Reply:
Sure, we can change it to 'ASSOCIATED_WITH'